apps/grav/webpack.grav.shared.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Grav-specific webpack config common to dev and prod.
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+
+const grav = {
+  entry: {
+    'app-grav': [
+      path.resolve(__dirname, 'index.js'),
+    ],
+  },
+  // These will be explicitly be provided OUTSIDE the bundle via a <script> tag in the HTML
+  externals: {
```

Comment:
Does grav include jquery & lodash by default, or are we doing that in our base grav templates already? I'm not seeing them actually included anywhere. otherwise i have no nits, everything looks good 👍